### PR TITLE
Make inventory square on all screen sizes

### PR DESCRIPTION
**Pull request by slwilliams**
**Preview here:** https://pr-426.lba2remake.net

### CODE REVIEW

**Review by xesf — APPROVED** [review]

File: src/ui/styles/inventory.scss [hunk 1]
```diff
@@ -49,6 +47,19 @@
     align-items: center;
 }
 
+@media screen and (orientation:landscape) { 
+    .inventory {
+        width: 115vh;
+        height: 98vh;
```
Comment (agrande) [inline src/ui/styles/inventory.scss:53]: Nice thinking to use vh as a unit for this!

**Review by agrande — APPROVED** [review]

**Comment by agrande** [comment]
If I'm not mistaken, the horizontal spacing between cells in the original is wider than the vertical.
Let me check this...

**Comment by agrande** [comment]
Also, it doesn't work properly in the editor due to vh units being relative to the whole window.

**Comment by agrande** [comment]
> If I'm not mistaken, the horizontal spacing between cells in the original is wider than the vertical.
> Let me check this...

My bad, it's the same spacing:
image
It's not properly aligned in the original, though! 😱

File: src/ui/styles/inventory.scss (hunk 1 above)
Comment (slwilliams) [reply src/ui/styles/inventory.scss:53]: yea I was glad this worked out well because otherwise it was going to be a little bit hacky.

**Comment by slwilliams** [comment]
Ack on the editor not working bit I think that should be easy to fix but I've advanced the branch I'm using locally so I'll do this in a follow up PR.

**Pull request merged**